package-lock.json

```diff
@@ -38,14 +35,12 @@
         "onetime": {
           "version": "1.1.0",
           "resolved": "https://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
-          "integrity": "sha1-ofeDj4MUxRbwXs78vEzP4EtO14k=",
-          "dev": true
+          "integrity": "sha1-ofeDj4MUxRbwXs78vEzP4EtO14k="
```

Comment:
I wonder why I have all these changes. I started from a clear install

Reply:
You moved `cypress` from dev to regular dependencies. That might be it :)

Reply:
Good catch :) Thanks